src/lib/rest/rest.cpp

```diff
@@ -835,6 +834,14 @@ int servicePathSplit(ConnectionInfo* ciP)
     LM_I(("Service Path %d: '%s'", ix, ciP->servicePathV[ix].c_str()));
   }
 
+  //
+  // stringSplit destroys ciP->servicePath.
```

Comment:
Wish it was that easy. That was my first attempt, of course
From httpRequestSend() we don't have that information, unfortunately.